src/components/icons/icons.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { SCVWrapperVaraints } from "./SVGWrapper/SVGWrapper.css";
+import * as icons from "./index";
+
+const macawIcons = Object.values(icons);
+
+export default {
+  title: "components/Icons",
+};
+
+export const Default = () =>
+  ["Small", "Medium", "Large"].map((size) => (
+    <>
+      <h3>{size}</h3>
+      <div
+        style={{
+          display: "flex",
+          alignItems: "center",
+          flexDirection: "row",
+          flexWrap: "wrap",
+        }}
+      >
+        {macawIcons.map((Icon, idx) => (
+          <div
+            style={{
+              margin: "10px",
+            }}
+            key={idx}
+          >
+            <Icon size={size.toLowerCase() as keyof SCVWrapperVaraints} />
```

Comment:
**suggestion**: move those size to their own variable

You can have `const sizes=['small','medium','large'] as const` and do not cast strings to `SCVWrapperVaraints` here